src/components/ToDoList/TodoWrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CustomTypeHooksType, TodoWrapperType } from './types';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+import taskManagementHandler from '../../hooks/CustomTaskHooks';
+
+function TodoWrapper(props: TodoWrapperType): JSX.Element {
+    const { todoTasks } = props;
+    const [
+        tasks,
```

Comment:
It looks like you're pulling tasks from two places. The hook and the props.

Reply:
Done